docs/README-develop-beaker-scripts.md

```diff
@@ -989,7 +990,7 @@ The keys are all optional. Most test cases will typically only use `:desc` and `
 | `:platform`       | A regexp pattern to match against the agent's product-id value. This is used to skip tests that don't support a feature or parameter. This key can be specified as a common top-level key or on a test-by-test basis.
 | `:preclean`       | The resource name used to remove a configuration as part of testbed pre-cleanup. The resource does not have to be the same as the one being tested; for example, the `'cisco_bgp_af'` test might use `'cisco_bgp'` for `:preclean` if a full bgp cleanup is needed instead of just removing the `'cisco_bgp_af'` configuration. This key can be specified as a common top-level key or on a test-by-test basis.
 | `:resource`       | A hash of expected values from the `puppet resource` command. This is only needed when the expected values differ from the values defined by `:manifest_props`; for example: default testing may specify a value of `'default'` in the manifest but an expected value of `'true'` for the `puppet resource` command.
-| `:ensure`         | The ensure value: `'present'` or `'absent'`. If not specified the test case will default to `'present'`.
+| `:ensure`         | The ensure value: `'present'` or `'absent'`. If not specified the test case will default to `'present'`. Also see Top-level key `:ensure`.
```

Comment:
They're related so I thought it would be helpful to keep it, although I should have used the new name.